pyproject.toml

```diff
@@ -1,3 +1,31 @@
+[project]
+name = "s2geometry"
+description = "Computational geometry and spatial indexing on the sphere"
+authors = [
+    { name = "Brian Miles", email = "selimnairb@gmail.com" },
+]
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: POSIX",
+    "License :: OSI Approved :: Apache Software License",
+]
+dynamic = [
+    "version",
```

Comment:
How is this different from the explicit version from `setup.cfg`?

Reply:
this uses `setuptools_scm` to derive the project version from the Git tag (or use a `git describe`-like string for the version in between tagged commits). Just one less place to update the version when making a release